### PR TITLE
@task PDA-1247 DELETE /items/name/{itemName} - build graphQL query

### PR DESCRIPTION
**Pull request by jchen-parasoft**

### CODE REVIEW

File: src/test/java/com/parasoft/demoapp/graphql/ItemGraphQLDataFetcherTest.java [hunk 1]
```diff
@@ -33,17 +39,39 @@ public class ItemGraphQLDataFetcherTest {
...
+
+    @Rule
+    public TestName testName = new TestName();
+
     @Before
     public void setUp() {
         graphQLTestTemplate.getHeaders().clear();
     }
 
+    @Before
```
Comment (yshi-parasoft) [inline src/test/java/com/parasoft/demoapp/graphql/ItemGraphQLDataFetcherTest.java:61]: Please add one more test case for empty item name.
